Makefile

```diff
@@ -5,7 +5,6 @@
 #  LINUX: Path to compiled Linux kernel tree.
 #  DEBIAN: Path to a pre-baked Debian image.
 
-RV64_PREFIX ?= riscv64-unknown-elf-
```

Comment:
Rebase artifact? Seems like it got deleted by accident?

Reply:
fixed